package.json

```diff
@@ -130,7 +130,7 @@
     "leaflet.heat": "0.2.0",
     "less": "2.7.1",
     "less-loader": "4.0.5",
-    "lodash": "3.10.1",
+    "lodash": "git://github.com/kobelb/lodash.git#3.10.2-npm",
```

Comment:
You might just be testing this for now, but ultimately we'll need to publish this to the elastic org on npm and use that. With URL references, yarn has to pull down the remote package every time you bootstrap in order to resolve the packages, whereas with npm repo packages it will determine whether it needs to pull based on the file system first.

Take a page from the hapi upgrade where the forked versioned it based on the scheme `<original version>-kibana1` https://github.com/elastic/kibana/pull/21707/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R64

Reply:
Using the hard-coded git link was just for testing, I'll be pushing to a package before we actually switch to this.